applications/incompressible_navier_stokes/flow_past_cylinder/include/grid.h

```diff
@@ -1018,20 +1018,42 @@ double const h_x_2 = (L - X_2) / nele_x_right;
 double const h_x_1 = D / nele_x_middle_middle;
 double const h_x_0 = (X_1 - X_0) / nele_x_left;
 
+void
+do_create_coarse_triangulation(dealii::Triangulation<2> &        tria,
```

Comment:
The code is nicely structured when providing a separate function for creating a hyper-rectangle. 

Could you give the function a more specific name? e.g. `ExaDG::create_subdivided_hyper_rectangle()`

This function could be implemented at a more central place in ExaDG, since this is potentially relevant for many other applications as well, see e.g. the function `ExaDG::create_periodic_box()` in the `grid` folder.

Reply:
> This function could be implemented at a more central place in ExaDG, since this is potentially relevant for many other applications as well, see e.g. the function `ExaDG::create_periodic_box()` in the `grid` folder.

I agree. I will rename it to the suggested name and leave it here for now. Because although the renaming is an improvement, I don't think it is the best name, since it is not clear from the name (at least for me) that this function is capable of creating both simplex and hyper-rectangle meshes.